x/ethbridge/types/test_common.go

```diff
@@ -12,35 +12,43 @@ import (
 )
 
 const (
-	TestAddress            = "cosmos1gn8409qq9hnrxde37kuxwx5hrxpfpv8426szuv"
-	TestValidator          = "cosmos1xdp5tvt7lxh8rf9xx07wy2xlagzhq24ha48xtq"
-	TestNonce              = 0
-	TestEthereumAddress    = "0x7B95B6EC7EbD73572298cEf32Bb54FA408207359"
-	AltTestEthereumAddress = "0x7B95B6EC7EbD73572298cEf32Bb54FA408207344"
-	TestCoins              = "10ethereum"
-	AltTestCoins           = "12ethereum"
+	TestEthereumChainID       = 3
+	TestBridgeContractAddress = "0xC4cE93a5699c68241fc2fB503Fb0f21724A624BB"
+	TestAddress               = "cosmos1gn8409qq9hnrxde37kuxwx5hrxpfpv8426szuv"
+	TestValidator             = "cosmos1xdp5tvt7lxh8rf9xx07wy2xlagzhq24ha48xtq"
+	TestNonce                 = 0
+	TestSymbol                = "eth"
+	TestTokenContractAddress  = "0x0000000000000000000000000000000000000000"
+	TestEthereumAddress       = "0x7B95B6EC7EbD73572298cEf32Bb54FA408207359"
+	AltTestEthereumAddress    = "0x7B95B6EC7EbD73572298cEf32Bb54FA408207344"
+	TestCoins                 = "10ethereum"
+	AltTestCoins              = "12ethereum"
 )
 
 //Ethereum-bridge specific stuff
 func CreateTestEthMsg(t *testing.T, validatorAddress sdk.ValAddress) MsgCreateEthBridgeClaim {
 	testEthereumAddress := NewEthereumAddress(TestEthereumAddress)
-	ethClaim := CreateTestEthClaim(t, validatorAddress, testEthereumAddress, TestCoins)
+	testContractAddress := NewEthereumAddress(TestBridgeContractAddress)
+	testTokenAddress := NewEthereumAddress(TestTokenContractAddress)
+	ethClaim := CreateTestEthClaim(t, testContractAddress, testTokenAddress, validatorAddress, testEthereumAddress, TestCoins)
 	ethMsg := NewMsgCreateEthBridgeClaim(ethClaim)
 	return ethMsg
 }
 
-func CreateTestEthClaim(t *testing.T, validatorAddress sdk.ValAddress, testEthereumAddress EthereumAddress, coins string) EthBridgeClaim {
+func CreateTestEthClaim(t *testing.T, testContractAddress EthereumAddress, testTokenAddress EthereumAddress, validatorAddress sdk.ValAddress, testEthereumAddress EthereumAddress, coins string) EthBridgeClaim {
```

Comment:
all tests should begin with `Test...`

Reply:
ditto, what are we testing here?

Reply:
@musnit

Reply:
This is not a test, it is a helper used in other tests to create test eth claims.